### PR TITLE
Refactor Landing Page

### PR DESCRIPTION
**Pull request by mde3**
## Tasks

- [x] Switch up to next 14
- [x] Update daisyUI from v3.5.1 to v4.4.20
- [x] Refactor code
- [x] Cleanup files 

## Reasons
- Previous next version wasn't supporting alata font from goole fonts, also overall next14 is more stable

### CODE REVIEW

File: apps/website/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,26 @@
 {
-  "extends": "tsconfig/nextjs.json",
   "compilerOptions": {
```
Comment (Emmanuel-Melon) [inline apps/website/tsconfig.json:2]: Most of these options are defined in the base config file in packages.

Comment (mde3) [reply apps/website/tsconfig.json:2]: Alright

**Review by eng-atwijukire — APPROVED** [review]

**Comment by eng-atwijukire** [comment]
@mde3 for further enhancement checkout a new-feature-branch from develop

**Comment by mde3** [comment]
@TrevorNyan Noted!

**Pull request merged**